backend/parse.js

```diff
@@ -122,3 +121,55 @@ export default function (data) {
     // TODO: keys geojson and raw are currently optional
     return {columnnames, ncols, nrows, rows, geojson, raw: data};
 }
+
+
+export function parseES(data) {
```

Comment:
it might be nice for this to be a little bit more verbose, like `parseElasticsearch`